PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
+    
+    public class Application {
+        // TODO: implement me
+    }
+    
+    public class Device {
+        
+        /// The platform name of the device i.e. "iPhone1,1" or "iPad3,6"
+        static let platform: String = {
```

Comment:
Good point. So refactoring wise we should add something like a `current` singleton which then returns an Object with all the current values. This way we cloud instantiate a Device with mocked values.